src/device_config.rs

```diff
@@ -788,11 +798,19 @@ impl DeviceConfig {
         *date_time_utc >= start_time && *date_time_utc <= end_time
     }
 
+    pub fn is_audio_device(&self) -> Option<bool>{
```

Comment:
Why does this not just return a bool?  I see you unwrapping it everywhere it's used.